.travis.yml

```diff
@@ -7,7 +7,7 @@ addons:
   apt:
     sources: &add-sources
     - sourceline: 'ppa:beineri/opt-qt571-trusty'
-    - sourceline: 'ppa:beineri/opt-qt-5.10.1-trusty'
```

Comment:
Apart from the proposed removed of `qt510speech` do we need/want to revert to 5.9 - or was 5.10 the possible cause of whatever problems there was/is with TTS? This comment applies to all the currently proposed changes in **this** file.